packages/xs-vat-worker/test/test-boot-lockdown.js

```diff
@@ -37,3 +40,30 @@ test('bootstrap to SES lockdown', async t => {
   await vat.close();
   t.deepEqual(['["function","function"]'], messages);
 });
+
+test('child compartment cannot access start powers', async t => {
+  const bootScript = await dist('bootstrap.umd.js');
+  const messages = [];
+  async function handleCommand(message) {
+    messages.push(decoder.decode(message));
+    return new Uint8Array();
+  }
+  const vat = xsnap({ ...xsnapOptions, handleCommand });
+  await vat.evaluate(bootScript);
+
+  const script = await fs.promises.readFile(
+    path.join(__filename, '..', 'escapeCompartment.js'),
+    'utf-8',
+  );
+  await vat.evaluate(script);
+  await vat.close();
+
+  t.deepEqual(messages, [
+    'hello from child',
+    'fo is function Object() { [native code] }',
+    'f is function Function() { [native code] }',
+    'err was TypeError: Not available',
+    'did evaluate',
+    `child compartment saw 'null'`,
+  ]);
```

Comment:
This seems brittle in the face of change, having to line up the expected outcomes with the send calls in the script above.

Maybe we can reframe this in terms of an assertion utility that bridges the gap. The `handleCommand` method inside the vat might evaluate the argument and send a string back. Then, an assertion function in the test would look like:

```js
t.equal(await query('{}.__proto__.constructor'), 'function Object() { [native code] }');
```

Reply:
The `TypeError: Not available` message is the point of the test.

The others are left over from flailing around with single-bit diagnostics from `xsnap`; I pruned them.

Reply:
I wonder whether the `Not available` error message is (intended to be) part of the specified behavior of SES. Brian and I found it in the source code, but if someone tweaks it, is it good or bad that this test would fail?

Reply:
Error types are standardized, but not error messages. It might be sufficient to verify that this effects a TypeError.